docs/3.x.x/guides/database.md

```diff
@@ -0,0 +1,296 @@
+# Database
+
+Strapi gives you options in which database you can use. Strapi currently supports the **PostgreSQL**, **MongoDB**, **SQLite**, **MySQL** and
+**MariaDB** databases. The following documentation covers how to install these databases locally (for development purposes) and on various hosted or cloud server solutions (for staging or production purposes).
+
+(Deploying **Strapi** itself is covered in the [Deployment Guide](/3.x.x/guides/deployment.html).)
+
+**Table of contents:**
+
+- [SQLite Installation](#sqlite-installation)
+- [MongoDB Installation](#mongodb-installation)
+
+
+---
+
+## SQLite Installation
+
+SQLite is the default [Quick Start](/3.x.x/getting-started/quick-start.html) database and recommended to quickly deploy an app locally.
```

Comment:
Should add a note that SQLite will not work with Heroku and is typically used to prototype. There may be some performance limitations depending on deployment type and location.

Reply:
@derrickmehaffy - Good point and I added a Note specifically about this.  Thank you.